domain/secret/state/state.go

```diff
@@ -153,7 +153,7 @@ func (st State) CheckApplicationSecretLabelExists(ctx domain.AtomicContext, appU
 	var exists bool
 	err := domain.Run(ctx, func(ctx context.Context, tx *sqlair.TX) error {
 		var err error
-		exists, err = st.checkApplicationSecretLabelExists(ctx, tx, appUUID, label)
+		exists, err = st.checkApplicationSecretLabelExists(ctx, tx, appUUID, label, "")
```

Comment:
question: The service package caller of `CheckApplicationSecretLabelExists` does have a URI, why not use it here? Same for `CheckUnitSecretLabelExists ` and `CheckUserSecretLabelExists `.